src/tasks.py

```diff
@@ -63,30 +67,48 @@ def command(
     """
     input_files = get_input_files(pipe_result, input_files or [])
     output_files = []
-    base_command = ["<REPLACE_WITH_COMMAND>"]
-    base_command_string = " ".join(base_command)
 
-    for input_file in input_files:
+    task_report = Report("SSH log analyzer report")
+
+    try:
+        log_year = int(task_config.get('log_year'))
+    except (TypeError, ValueError):
+        log_year = None 
+
+    ssh_analysis_task = LinuxSSHAnalysisTask(log_year=log_year)
+    analyzer_output_priority = Priority.LOW
+
+    df = ssh_analysis_task.read_logs(input_files=input_files)
+    if df.empty:
+        task_report.summary = "No SSH authentication events in input files."
+
+    else:
+
+        # 01. Brute Force Analyzer
+        (result_priority, result_summary, result_markdown) = (
+            ssh_analysis_task.brute_force_analysis(df)
+        )
+        if result_priority < analyzer_output_priority:
+            analyzer_output_priority = result_priority
+        task_report.summary = result_summary
+
         output_file = create_output_file(
             output_path,
-            display_name=input_file.get("display_name"),
-            file_extension="<REPLACE_WITH_FILE_EXTENSION>",
-            data_type="<[OPTIONAL]_REPLACE_WITH_DATA_TYPE>",
+            display_name="linux_ssh_analysis",
+            extension=".md",
+            data_type="openrelik:ssh:report",
         )
-        command = base_command + [input_file.get("path")]
-
-        # Run the command
-        with open(output_file.path, "w") as fh:
-            subprocess.Popen(command, stdout=fh)
+        with open(output_file.path, "w") as outfile:
+            outfile.write(result_markdown)
 
         output_files.append(output_file.to_dict())
 
-    if not output_files:
-        raise RuntimeError("<REPLACE_WITH_ERROR_STRING>")
+        if not output_files:
```

Comment:
You always create an output_file above so I do not think this will ever trigger that output_file is None or empty.